js/components/controlBar.js

```diff
@@ -313,18 +319,10 @@ var ControlBar = createReactClass({
   },
 
   populateControlBar: function() {
-    var playIcon;
-    var playPauseAriaLabel;
-    if (this.props.playerState === CONSTANTS.STATE.PLAYING) {
-      playIcon = 'pause';
-      playPauseAriaLabel = CONSTANTS.ARIA_LABELS.PAUSE;
-    } else if (this.props.playerState === CONSTANTS.STATE.END) {
-      playIcon = 'replay';
-      playPauseAriaLabel = CONSTANTS.ARIA_LABELS.REPLAY;
-    } else {
-      playIcon = 'play';
-      playPauseAriaLabel = CONSTANTS.ARIA_LABELS.PLAY;
-    }
+    var playButtonDetails = Utils.getPlayButtonDetails(this.props.playerState);
```

Comment:
just a note, but should we make the way we spell button consistent here?
You have playButtonDetails and playBtnTooltip.  The length of the var is about the same if they are consistent.

Reply:
Ah I moved these lines, so that is how they were spelled before. Will make this change.